proxied.d.ts

```diff
@@ -861,12 +876,20 @@ export interface Typegram<F> {
       chat_id: number | string;
     }): ChatMember[];
 
-    /** Use this method to get the number of members in a chat. Returns Int on success. */
+    /**
+     * @deprecated
```

Comment:
Maybe highlight what to use instead?
```suggestion
     * @deprecated use getChatMemberCount instead
```